src/components/LayersPanel/LayersPanel.js

```diff
@@ -27,6 +27,23 @@ class LayersPanel extends React.PureComponent {
       window.docViewer.updateView();
     }
   }
+  componentDidMount() {
+    core.addEventListener('layersUpdated', this.onLayersUpdated.bind(this));
+  }
+
+  componentWillUnmount() {
+    core.removeEventListener('layersUpdated', this.onLayersUpdated.bind(this));
+  }
+
+  onLayersUpdated() {
```

Comment:
Can we follow the convention in other components to use class properties?

```javascript
onLayersUpdated = () => {...}
```

And then we don't need to `.bind(this)`